### PR TITLE
Update graham2021levit_1.md

### PR DESCRIPTION
**Pull request by at-ray-o**

### CODE REVIEW

File: _summaries/graham2021levit_1.md [hunk 1]
```diff
@@ -1,19 +1,27 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: graham2021levitt
```
Comment (zhaoyue-zephyrus) [inline _summaries/graham2021levit_1.md:4]: typo: graham2021levitt -> graham2021levit

File: _summaries/graham2021levit_1.md [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+The paper attempts to combine, convolutional networks and training principle from CNNs with vision transformers.
+![image](https://user-images.githubusercontent.com/13065170/136635751-c21a6e5d-c6ce-4813-bda9-4f3ef884e265.png)
+![image](https://user-images.githubusercontent.com/13065170/136635765-4209a6b7-7e90-436c-a96f-2cb958586416.png)
```
Comment (zhaoyue-zephyrus) [inline _summaries/graham2021levit_1.md:14]: You may add a few words (in bullet e.g.) to describe what design in the figures reflects what principle from CNN.

File: _summaries/graham2021levit_1.md [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+
+The transformer layers are stacked like a ResNet. The ResNet extracts features from the transformer layers and the gradients are back-propagated. Thus all CNN operations can be applied on the intermediate representations.
```
Comment (zhaoyue-zephyrus) [inline _summaries/graham2021levit_1.md:16]: This is not quite clear to me. What does "The ResNet extracts feature from the transformer layers" mean? CNN operations are place on the bottom layers (see Fig 4 on the paper).

File: _summaries/graham2021levit_1.md [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
+![image](https://user-images.githubusercontent.com/13065170/136635784-094d7434-e487-40a4-b3fa-a14a7eb44a4e.png)
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* CNN strategies applied to vision transformers
+* Convolutional operations applied to transformer stacks
+* Results in much faster object detection
```
Comment (zhaoyue-zephyrus) [inline _summaries/graham2021levit_1.md:27]: Is it object detection or merely image classification? Is the final 4x4 feature map sufficient for object detection?

**Review by zhaoyue-zephyrus — COMMENTED** [review]
This draft contains the main messages of the paper with illustrative figures. However, some details are missing and some important architectural designs are needed to be made clear.

**Review by zhaoyue-zephyrus — APPROVED** [review]
